contracts/PartnerConfiguration/PartnerConfiguration.sol

```diff
@@ -90,6 +98,8 @@ contract PartnerConfiguration is IPartnerConfiguration, Ownable {
         }
 
         _minLength = minLength;
+
+        emit MinLengthChanged(preModifiedValue, _minLength);
```

Comment:
I talked to Ulas about events being fires before or after and he said it is not important. But if you did it at the begiining in the other PR, just follow the same principle here